src/ROUTES.ts

```diff
@@ -943,9 +943,10 @@ const ROUTES = {
         getRoute: (threadReportID: string) => `r/${threadReportID}/duplicates/review/tax-code` as const,
     },
     TRANSACTION_DUPLICATE_REVIEW_DESCRIPTION_PAGE: {
-        route: 'r/:threadReportID/duplicates/confirm',
-        getRoute: (threadReportID: string) => `r/${threadReportID}/duplicates/confirm` as const,
+        route: 'r/:threadReportID/duplicates/review/description',
+        getRoute: (threadReportID: string) => `r/${threadReportID}/duplicates/review/description` as const,
     },
+
```

Comment:
I don't want to block on this, but once we do any other polish issues, we should remove this extra line

```suggestion
```